contents/articles/2017-01_saliency-in-vr/index.md

```diff
@@ -0,0 +1,21 @@
+--- 
+title: "Saliency in VR: How do people explore virtual environment"
+date: Mon Jan 09 2017 11:45:18 GMT+0700 (ICT) 
+author: Pupil Dev Team 
+subtitle: "Vincent et al., proposes new methods to learn and predict time-dependent saliency in VR to make eye tracking expendable..."
+featured_img: "../../../../media/images/blog/vr_saliency.png"
+featured_img_thumb: "../../../../media/images/blog/thumb/vr_saliency.png"
+---
+
+In their recent research paper [Saliency in VR: How do people explore virtual environments?], Vincent Sitzmann et al., argue that viewing behavior in VR environments is much more complex than on conventional displays due to the technology of interactions in kinematics that makes VR possible.
+
+<img src="../../../../media/images/blog/vr_saliency.png" class='Feature-image u-padBottom--1 u-padTop--1' alt="Ground Truth Saliency Map">
+Image Source: [Saliency in VR: How do people explore virtual environments?](https://arxiv.org/pdf/1612.04335.pdf)
+
+To further understand the view behavior and saliency in VR, Vincent Sitzmann et al. have collected a dataset that records gaze data and head orientation from users oberserving omni-directional stereo panoramas using an Oculus DK2 head-mounted display with Pupil Labs's [Oculus Rift DK2 add-on cup](https://pupil-labs.com/store/#vr-ar). 
+
+The dataset reveals gaze and head orientation do have a correlation together in VR with some cases of differences. Based on the data, Sitzmann and his colleagues propose new methods to learn and predict time-dependent saliency in VR. The collected data is a first step towards building saliency models specifically tailored to VR. If successful, these VR saliency models could serve as a method to approximate gaze movements from head movement and image information alone.
```

Comment:
The dataset reveals that gaze and head orientation are correlated and argue that they can be used to build more accurate saliency maps for VR environments.